Formula/vulkan-tools.rb

```diff
@@ -55,12 +56,12 @@ def install
       "-DBUILD_CUBE=ON",
       "-DBUILD_VULKANINFO=ON",
       "-DINSTALL_ICD=OFF", # we will manually place it in a nonconflicting location
-      "-DGLSLANG_INSTALL_DIR=#{Formula["glslang"].prefix}",
-      "-DVULKAN_HEADERS_INSTALL_DIR=#{Formula["vulkan-headers"].prefix}",
-      "-DVULKAN_LOADER_INSTALL_DIR=#{Formula["vulkan-loader"].prefix}",
+      "-DGLSLANG_INSTALL_DIR=#{Formula["glslang"].opt_prefix}",
+      "-DVULKAN_HEADERS_INSTALL_DIR=#{Formula["vulkan-headers"].opt_prefix}",
+      "-DVULKAN_LOADER_INSTALL_DIR=#{Formula["vulkan-loader"].opt_prefix}",
```

Comment:
Shouldn't be needed -- this evaluates to the same thing. It's only inside the formula that `prefix` and `opt_prefix` are different.

Reply:
That's interesting to know, because making that change is precisely what made the difference for me.

Reply:
Without it, and with `molten-vk` HEAD installed instead of stable, I get this error in the logs when reinstalling `vulkan-tools` from source:
```
gmake[2]: *** No rule to make target '/usr/local/Cellar/molten-vk/1.2.3/share/vulkan/icd.d/MoltenVK_icd.json', needed by 'CMakeFiles/MoltenVK_icd-staging-json'.  Stop.
```

Reply:
With this change, the error goes away and the formula builds successfully.

Reply:
Hm, weird. Ok, let's merge this anyway.